web/src/hashchange.js

```diff
@@ -77,6 +77,14 @@ function show_home_view() {
     //
     // We only allow the primary recommended options for home views
     // rendered without a hash.
+
+    if (page_params.is_spectator) {
+        // We show recent view to spectators regardless of default.
+        maybe_hide_inbox();
+        recent_view_ui.show();
+        return;
+    }
```

Comment:
I think the more normal way to do this would be to force the value of the `home_view` property in the API.

Reply:
Makes sense.

Reply:
Updated!